pages/osx/rubocop.md

```diff
@@ -0,0 +1,31 @@
+# rubocop
+
+> Lint Ruby files.
+
+- Check all files in the current directory:
+
+`rubocop`
+
+- Check specific files or directories:
+
+`rubocop {{path_1}} {{path_2}}`
+
+- Write output to file:
+
+`rubocop --out {{file}}`
+
+- View list of cops:
```

Comment:
We should also explain what a cop is.

Reply:
What's a good pattern for this? I can either:

1. Work it into the description at the top;
2. Do it inline for the first example that mentions cops; or
3. Add a separate section that explains it (are there any other pages that do something like this?).

Reply:
inline is fine.